djangoevents/unifiedtranscoder.py

```diff
@@ -29,10 +29,10 @@ def serialize(self, domain_event):
         assert isinstance(domain_event, DomainEvent)
 
         event_data = domain_event.__dict__.copy()
-        event_data.pop('domain_event_id')
-        event_data.pop('entity_id')
-        event_data.pop('entity_version')
-        event_data.pop('metadata')
+        event_data.pop('domain_event_id', None)
+        event_data.pop('entity_id', None)
+        event_data.pop('entity_version', None)
+        event_data.pop('metadata', None)
```

Comment:
`pop` is used here to remove keys, not to get them. I opened a new pull request that explicitly filters keys: #7.